packages/esm-admin-openconceptlab-app/src/import/import.resource.ts

```diff
@@ -0,0 +1,42 @@
+import { openmrsFetch } from '@openmrs/esm-framework';
+import { Subscription } from 'rxjs';
+import useSWR from 'swr';
+
+import { Import } from '../types';
+
+export function useSubscription() {
+  const { data, error, isValidating } = useSWR<{ data: { results: Subscription[] } }, Error>(
+    '/ws/rest/v1/openconceptlab/subscription?v=full',
+    openmrsFetch,
+  );
+
+  return {
+    data: data?.data?.results[0],
+    isLoading: !data && !error,
+    isError: error,
+    isValidating,
+  };
+}
+
+export async function startImportWithSubscription(abortController?: AbortController) {
+  const url = '/ws/rest/v1/openconceptlab/import';
+  return openmrsFetch<Import>(url, {
+    method: 'POST',
+    body: {},
+    headers: {
+      'Content-Type': 'application/json',
+    },
```

Comment:
It's a little weird to be setting a `Content-Type` in a header when there's no body...

Reply:
When using this POST request, the backend requires an empty request body. Otherwise it throws a 400 error.
<img width="992" alt="image" src="https://user-images.githubusercontent.com/27498587/177829441-9488c3f9-3dea-4d40-90d9-a83537a17f24.png">

After adding an empty body, if we don't set the `Content-Type`, the request is sent with `text/plain` content type in default. But then, the backend throws a 500 error saying that it doesn't support `text/plain` content type. 

<img width="992" alt="image" src="https://user-images.githubusercontent.com/27498587/177828040-ef598179-4387-49f8-a91d-ab6f2119b69d.png">

So I manually set the `Content-Type` as `application/json`

Reply:
Ok... fair... so it's the backend that's being weird here. Fair enough!